executor/extension/statedb/temporary_state_prepper.go

```diff
@@ -47,6 +47,15 @@ type temporaryInMemoryStatePrepper struct {
 	extension.NilExtension[txcontext.TxContext]
 }
 
+func (p *temporaryOffTheChainStatePrepper) PreRun(executor.State[txcontext.TxContext], *executor.Context) error {
+	chainCfg, err := utils.GetChainConfig(utils.EthereumChainID)
```

Comment:
This line looks incorrect. This extension is also used by other tools beside eth-tests.

Reply:
`aida-vm` runs substate of fantom chain, it needs proper berlin and london block height of fantom to operate correctly. Setting the chain config to EthereumChainID still looks incorrect.

Reply:
You are right. I changed the code alltogether to remove redundancy, as we create `ChainCfg` in `configContext`